tests/unit/plugins/module_utils/test_query.py

```diff
@@ -96,7 +96,10 @@ def test_and_serialize_query(self):
             [{"description": ("LIKE", "email"), "short_description": ("LIKE", "SAP")}]
         )
 
-        assert result == "descriptionLIKEemail^short_descriptionLIKESAP"
```

Comment:
I would rather go with something like this:

    set(result.split("^")) == set(("descriptionLIKEemail", "short_descriptionLIKESAP"))

This is a bit more robust and easier to extend.